### PR TITLE
feat(MOPLAT-261): Sentence case on placeholder

### PR DESCRIPTION
**Pull request by patrinoua**
This PR resolves [MOPLAT-261]

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look on our [docs], or get in touch with us.

[app state migration]: /docs/adding_state_migrations.md
[feature flag]: /docs/developing_a_feature.md
[docs]: /docs/README.md

[MOPLAT-261]: https://artsyproduct.atlassian.net/browse/MOPLAT-261?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pvinis — DISMISSED** [review]
#squashongreen

**Review by AlicanAkyuz — APPROVED** [review]
👏

**Review by rajsam003 — APPROVED** [review]
Great @patrinoua. 
Our design team should also be aware of these changes. 😉

File: src/app/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tests.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ describe("OnboardingCreateAccount", () => {
 
     const nameInput = tree.root.findByProps({ testID: "nameInput" })
     nameInput.props.onChangeText("Full Name")
-    expect(nameInput.props.placeholder).toEqual("First and Last Name")
+    expect(nameInput.props.placeholder).toEqual("First and last Name")
```
Comment (pvinis) [inline src/app/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tests.tsx:74]: what about Name?

**Comment by damassi** [comment]
Can we be sure to update the PR with a full description? Links to JIRA are good, but each PR should contain a full description of the change 🙏

**Comment by gkartalis** [comment]
First of all great job and thanks for addressing that! 

Leaving one tiny thought here, wouldn't it be great if we had a helper function (or an [npm package](https://github.com/artsy/to-title-case) like @dimatretyak made a while back) with all the rules that we want to apply on each placeholder and go add this `once` in the palette components?

This will help devs not really care much about the copies and if we decide in the future to change the behavior of this all we need to do is just change the helper function!

Let me know your thoughts on this! 🤔

**Comment by gkartalis** [comment]
also @pvinis I think that #squashOnGreen doesn't work anymore in eigen, not sure why

**Comment by patrinoua** [comment]
Thank you all for the comments and suggestions 🙏

@damassi I have updated the description to provide more information.

@gkartalis About a more flexible change, I'd be happy to do it if we think there is a business case that we Might Want To Start Writing Like This Again, (OR MAYBE LIKE THIS?), however, this seems unlikely to me and therefore not a valuable time investment. 

@rajsam003 This is a design decision, here is the link (linked on the Jira ticket) 
https://www.notion.so/artsy/Sentence-Case-change-for-data-entry-field-placeholders-5edc233e8f844adc906117b55604a06c#00c5d74299944c1da4473500b2bb5920

**Pull request merged**